ibm/mas_devops/roles/suite_app_config/vars/manage.yml

```diff
@@ -107,9 +107,9 @@ mas_app_settings_server_bundles:
         isDefault: false
         isMobileTarget: false
         isUserSyncTarget: false
-        name: report
+        name: rpt
```

Comment:
Would this be a breaking change for anyone running the install over the top of an existing install?